package.json

```diff
@@ -11,10 +11,11 @@
   ],
   "description": "Single shot implementation of ridge regression in python for COINSTAC.",
   "dependencies": {
-    "freesurfer-parser": "0.0.2"
+    "freesurfer-parser": "0.0.2",
+    "numeric": "^1.2.6"
```

Comment:
`numeric` must be added to _package.json_ such that other developers and COINSTAC itself has the dependency. This is a Node.js convention.